charts/backstage/templates/deployment.yaml

```diff
@@ -26,8 +26,10 @@ spec:
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
 {{- end }}
-          image: "{{ template "backstage.image" .Values.image }}"
+          image: "{{ include "backstage.image" . }}"
```

Comment:
The default image value that points to the `ImageStream` does not resolve automatically. The annotation `alpha.image.policy.openshift.io/resolve-names: '*'` must be applied to the PodTemplate